o4/o4_pyforce.py

```diff
@@ -30,8 +30,9 @@ def __init__(self, *args):
         self.stderr = NamedTemporaryFile()
         if os.environ.get('DEBUG', ''):
             print(f'## p4', *self.args, file=sys.stderr)
-        self.pope = Popen(
-            ['p4', '-vnet.maxwait=60', '-G'] + self.args, stdout=PIPE, stderr=self.stderr)
+        self.pope = Popen(['p4', '-vnet.maxwait=60', '-G'] + self.args,
+                          stdout=PIPE,
+                          stderr=self.stderr)
```

Comment:
Is your yapf set up correctly and updated to the latest version? Looks like you're wrapping at 80

Reply:
Ah, yes it is. I get this wrapping with the latest version. Sigh.